wiki0/src/client/components/editpage.tsx

```diff
@@ -15,6 +15,14 @@ type RequestLogItem = {
 
 const API_BASE_URL = '/api/articles';
 
+const getParsedTokenString = (headers: string) => {
+  const parts = headers.split('Bearer ');
+  if (parts?.length === 2) {
+    return parts[1]?.replaceAll('"', '')?.replaceAll('}', '');
```

Comment:
Whats the token here? It's not base64?

Reply:
Oh that is super weird that it has a `}"` at the end. (Bug?)

Reply:
Maybe add a comment? `// TODO: Token building is probably buggy, will look into and clean up later`